ci/pkl-pipelines/pay-deploy/deploy-to-production.pkl

```diff
@@ -30,7 +31,13 @@ resources = new {
   pipeline_self_update.PayPipelineSelfUpdateResource("pay-deploy/deploy-to-production.pkl", "master")
   shared_resources_for_metrics.prometheusPushgatewayResource
   shared_resources.payCiGitHubResource
-  shared_resources.payGithubResourceWithBranch("pay-infra", "pay-infra", "master")
+  new PayResources.PayGitHubResource {
```

Comment:
I'm wondering whether we ought to create a `PayResources.PayInfraGitHubResource` since this is used 7 times throughout?

Reply:
Sounds like a grand idea :)